app/components/chart-pie.js

```diff
@@ -26,37 +27,57 @@ export default Component.extend({
     const width = get(this, 'width');
     const height = get(this, 'height');
     const displayTooltip = get(this, 'displayTooltip');
-    const hideTooltip = get(this, 'hideTooltip');
+    // const hideTooltip = get(this, 'hideTooltip');
     const radius = Math.min(width, height) / 2;
     const color = scaleOrdinal(schemeCategory10);
 
-    let t = transition().duration(2000).ease(easeLinear);
+    let t = transition().duration(500).ease(easeLinear);
 
+    /*let t2 = transition()
+          .ease(easeLinear)
+          .duration(2000)
+          .attrTween("d", tweenPie);
+    */
     let createArc = arc().innerRadius(0).outerRadius(radius);
     let createPie = pie().value(d => d.data).sort(null);
     let createLabelArc = arc().outerRadius(radius - 32).innerRadius(radius - 32);
 
-    let chart = svg.append('g').attr('transform', 'translate(' + (width / 2) +  ',' + (height / 2) + ')');
+    let chart = svg.append('g').attr('class', 'pie').attr('transform', 'translate(' + (width / 2) +  ',' + (height / 2) + ')');
     let path = chart.selectAll('path').data(createPie(dataOrArray)).enter()
+      .append('g').attr('class', 'slice')
       .append('path')
+      .attr('class', 'slicepath')
       .attr('d', createArc)
       .attr('stroke', '#FFFFFF')
       .attr('fill', d =>  color(d.data.label));
 
-    path.on('mouseover', d => displayTooltip(d.data));
-    path.on('mouseout', d => hideTooltip(d.data));
+    chart.selectAll('path.slicepath').transition()
+      .ease(easeLinear)
+      .duration(1000)
+      .attrTween("d", tweenPie);
+
+    function tweenPie(b) {
+      b.innerRadius = 0;
+      var i = interpolate({startAngle: 0, endAngle: 0}, b);
+      return function(t) { return createArc(i(t)); };
+    }
+
+    path.on('mouseover', function(d, index, items) {displayTooltip(d.data, items[index], createLabelArc.centroid(d));});
+    //path.on('mouseout', d => hideTooltip());
```

Comment:
Commented out code has been removed.  We can wait to merge if you would like.  I am still working on the same branch and do not have the latest updates for master. If I can keep working on my current branch, then I don't see a need to merge.  I'll leave the decision up to you. Thanks!